packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -109,6 +108,7 @@ function getFlowVersion(options) {
 }
 
 function initializeFlow(projectPath, flowconfig, otherFlowTypedDefs) {
+  const flowTypedPath = path.join(projectPath, 'node_modules', '.bin', 'flow-typed');
```

Comment:
It should be, at least with yarn and npm 3+ (the builds are currently passing, they are at least further than usual!)